ops/cloud-deployment/monitoring-alerts/metrics-alert-rule.bicep

```diff
@@ -0,0 +1,103 @@
+@description('Alert Rule Name')
+param alertName string
+
+@description('Id of the application or resource this metric alert is for')
+param appId string
+
+@description('Action Group Name for alerts')
+param actionGroupName string
+
+@description('Action Group Resource Group Name for alerts')
+param actionGroupResourceGroupName string
+
+@allowed([
+  'Total'
+  'Maximum'
+  'Average'
+])
+@description('Allowed values for time Aggregation')
+param timeAggregation string
+
+
+@allowed([
+  'GreaterThan'
+  'LessThan'
+  'GreaterThanOrEqual'
+  'LessThanOrEqual'
+  'Equal'
+])
+@description('Allowed values for operator')
+param operator string
+
+@allowed([
+  'Microsoft.Web/sites'
+  'Microsoft.Sql/servers/databases'
+  'Microsoft.DocumentDB/databaseAccounts'
+])
+@description('Allowed values for targetResourceType')
+param targetResourceType string
+
+@allowed([
+  'Http5xx'
+  'storage_percent'
+  'ServiceAvailability'
+  'HealthCheckStatus'
+  'storage_percent'
+])
+@description('Allowed values for metricName')
+param metricName string
+
+@description('Values for severity 0 = Critical, 1 = Error, 2 = Warning, 3 = Informational, 4 = Verbose')
+param severity int
+
+@description('Values for threshold')
+param threshold int
+
+@description('Values for Evaluation Frequency')
+param evaluationFrequency string = 'PT15M' //Default to 15M for most Alerts, Servicevailability is an exception requires 1h+
```

Comment:
The thresholds and frequency of each of these can be adjusted as we move forward. For now I have them set to these defaults.